core/lib/crypto/src/merkle_tree/parallel_smt.rs

```diff
@@ -882,7 +882,7 @@ mod tests {
         for &(idx, value) in &test_vector[..] {
             let merkle_proof = tree.merkle_path(idx);
 
-            let hasher = TestHasher::default();
+            let hasher = TestHasher;
```

Comment:
@franciscotobar why this change was necessary for this PR?

Reply:
to fix the lint process